### PR TITLE
pnixon caption

### PR DESCRIPTION
**Pull request by pnixon-cartera**
Updating Slide for 0000-04 pnixon

### CODE REVIEW

**Review by kizzo-cartera — APPROVED** [review]
LGTM

**Comment by githubteacher** [comment]
This is where global communication happens.

File: _posts/0000-01-04-pnixon-cartera.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+True Life!
```
Comment (githubteacher) [inline _posts/0000-01-04-pnixon-cartera.md]: True true.

File: _posts/0000-01-04-pnixon-cartera.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
 
 ![steroidtocat](https://octodex.github.com/images/steroidtocat.png)
 {: .center}
```
Comment (githubteacher) [inline _posts/0000-01-04-pnixon-cartera.md:9]: What do you think about adding another caption below the picture?

**Review by githubteacher — DISMISSED** [review]
This looks really bad! Left a minor suggestion for you. 

👍 💯 🔥 🎆

File: _posts/0000-01-04-pnixon-cartera.md (hunk 1 above)
Comment (githubteacher) [inline _posts/0000-01-04-pnixon-cartera.md]:
```suggestion
Fake life!
```

Comment (pnixon-cartera) [reply _posts/0000-01-04-pnixon-cartera.md]: #Fake news, I like my version.

**Pull request merged**